openshift/player.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+ARGV=${@:2}
```

Comment:
what does this script do in openshift?

Reply:
It will simplify the pipeline code, improving maintanibility. It also allows you to run the same commands from your command line for debugging etc